pkg/asset/machines/platform.go

```diff
@@ -0,0 +1,48 @@
+package machines
+
+import (
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/asset/machines/aws"
+	awstypes "github.com/openshift/installer/pkg/types/aws"
+	"github.com/pkg/errors"
+)
+
+// platform holds information retrieved from the platform where it can
+// be shared between Master and Worker.
+type platform struct {
+	AWSZones []string
+}
+
+var _ asset.Asset = (*platform)(nil)
+
+// Dependencies returns all of the dependencies directly needed by the
+// platform asset.
+func (a *platform) Dependencies() []asset.Asset {
+	return []asset.Asset{
+		&installconfig.ClusterID{},
```

Comment:
This should be `&installconfig.InstallConfig{}`.

Reply:
> This should be `&installconfig.InstallConfig{}`.

[Obsolete now][1].

[1]: https://github.com/openshift/installer/pull/1212#issuecomment-469597418